README.rst

```diff
@@ -10,4 +10,174 @@ test base classes and documentation shared by many XBlocks.
 
 To test the utilities, run::
 
-    nosetests --with-coverage --cover-package xblockutils --cover-html
+    python run_tests.py --with-coverage --cover-package=xblockutils
+
+
+StudioEditableXBlockMixin
+-------------------------
+
+.. code:: python
+
+    from xblockutils.studio_editable import StudioEditableXBlockMixin
+
+This mixin will automatically generate a working ``studio_view`` form
+that allows content authors to edit the fields of your XBlock. To use,
+simply add the class to your base class list, and add a new class field
+called ``editable_fields``, set to a tuple of the names of the fields
+you want your user to be able to edit.
+
+.. code:: python
+
+    @XBlock.needs("i18n")
```

Comment:
`needs("i18n")` is a very common, useful decorator, since it's basically the only way for an XBlock to localize its translatable strings. It's not strictly required, but it's in this example code as a reminder of good practices.